app/pages/TrusteeLiveInTheUKPage.scala

```diff
@@ -28,6 +28,7 @@ override def path: JsPath = JsPath \ Trustees \ index \ toString
 
   override def toString: String = "liveInUK"
 
+  // TODO this is incomplete
```

Comment:
Completed as part of this ticket?

Reply:
No, we're going to have to have a broader look at this for all addresses and re-model how we're writing and reading addresses from mongo.